src/main/webapp/WEB-INF/views/newCustomerService.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Customer Service</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.2/css/bootstrap.css">
+    <link rel="stylesheet" href="newCustomerService.css">
+</head>
+
+    <body>
+    <div class="container-fluid">
+        <!-- NavBar Starts -->
+        <nav class="navbar navbar-expand-sm navbar-light bg-light" >
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo03" aria-controls="navbarTogglerDemo03" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <div class="collapse navbar-collapse" id="navbarTogglerDemo03 ">
+                <ul class="navbar-nav mr-auto mt-2 mt-lg-0">
+
+                    <li class="nav-item mx-2">
+                        <a class="nav-link" href="#">Product <span class="sr-only">(current)</span></a>
+                    </li>
+
+                    <li class="nav-item dropdown dmenu mx-2">
+                        <a class="nav-link dropdown-toggle" href="#" id="navbardrop" data-toggle="dropdown">
```

Comment:
for this dropdown's id, and the rest, why not follow something along the lines of "navbardrop1", "navbardrop2", etc?